types/validator_set_test.go

```diff
@@ -1232,21 +1232,21 @@ func TestValSetUpdatePriorityOrderTests(t *testing.T) {
 func verifyValSetUpdatePriorityOrder(t *testing.T, valSet *ValidatorSet, cfg testVSetCfg, nMaxElections int) {
 	// Run election up to nMaxElections times, sort validators by priorities
 	valSet.IncrementProposerPriority(tmrand.Int()%nMaxElections + 1)
-	origValsPriSorted := validatorListCopy(valSet.Validators)
```

Comment:
I don't think this is the reason for timeout https://app.circleci.com/pipelines/github/tendermint/tendermint/3849/workflows/6d241fc8-72a8-4a3b-b3fc-887a48518f82/jobs/114996. still investigating